### ISSUE TITLE
Can we apply this kind of method to give more efficient in the sales module?

### ISSUE THREAD

**Issue by izeblood** [opensourcepos/opensourcepos#2551]
![process!](https://user-images.githubusercontent.com/32444665/65569760-460cb280-df91-11e9-9373-993429da9dd0.jpg)

then the result ...
![2output](https://user-images.githubusercontent.com/32444665/65569922-eebb1200-df91-11e9-8ebb-82467667b2a3.jpg)

**Comment by SteveIreland** [comment]
This looks like interesting addition.  Since @daN4cat has implicitly approved it I'll take it on as an early 3.4 change unless someone else would rather do it themselves.

**Comment by SteveIreland** [comment]
Something to think about.  Maybe a user is search for an item with 3*xxx in the item description and isn't intending to place an order for an item with a quantity of 3.  I think an asterisk would be too common.

Any suggestions?   I'm toward a space-double forward slash-space i.e. "3 // upc", but that could be problematic considering cross-script issues.

**Comment by izeblood** [comment]
for me * is more convenient since its stance for x (multiplication operation) and very easy to key in

**Comment by WebShells** [comment]
I agree, maybe * is easier to input during sales through keypad, the nearest.
Please @SteveIreland keep in mind digital scale barcode format.
Thanks in advance!

**Comment by izeblood** [comment]
I hope this can be added at 3.4.0 release... Thanks in advance @SteveIreland  @WebShells

**Comment by izeblood** [comment]
any updates on this feature @SteveIreland
